docs/src/howto-guides/011_retrieve_jwt_token_from_cumulocity.md

```diff
@@ -0,0 +1,26 @@
+## How to retrieve a JWT Token to authenticate on Cumulocity
+
+## Overview
+
+In order to [authenticate HTTP requests on Cumulocity](https://cumulocity.com/guides/10.5.0/reference/rest-implementation/#authentication),
```

Comment:
The link is too old. I think this one is the new: https://cumulocity.com/api/#section/Authentication/JWT